source/playbooks/get-started-with-playbooks.rst

```diff
@@ -0,0 +1,55 @@
+Get started with Mattermost Playbooks
+=====================================
+
+Access Mattermost Playbooks by selecting **Playbooks** from the **Product list** |product-list| icon.
+
+.. |product-list| image:: ../images/products_E82F.svg
+  :height: 24px
+  :width: 24px
+  :alt: Navigate between Channels, Playbooks, and Boards using the Product list icon.
```

Comment:
This page may need to be added to the TOC.
Also the icon is not rendering correctly on preview.

Reply:
Thanks, Amy!

I will add it to the TOC today and the icon isn't rendering because it's part of a PR from Carrie that'll be merged today - the icon should render then.

Reply:
So it's dependent on https://github.com/mattermost/docs/pull/5391 being merged first.